components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/AbstractApplicationAuthenticatorTest.java

```diff
@@ -216,20 +216,30 @@ public void testGetClaimDialectURI() throws Exception {
         Assert.assertNull(abstractApplicationAuthenticator.getClaimDialectURI());
     }
 
-    @Test
-    public void testSetTenantDomainToUserName() throws Exception {
+    @Test(dataProvider = "usernameProvider")
```

Comment:
Wrong data provider here, hence should be:
```suggestion
    @Test(dataProvider = "userProvider")
```
I think that's why the build is failing too.